controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
     {
+        $defaultLanguageId = $this->getDefaultLanguageId();
+
+        foreach ($fields as $name => $value) {
+            if ('name_' . $defaultLanguageId === $name) {
+                if (is_string($value) && empty(trim($value))) {
+                    $this->errors[] = $this->module->getTranslator()->trans(
+                        'The "Name of the link block" is required', array(), 'Modules.LinkList');
+
+                    return false;
+                }
+            }
+        }
+
         return true;
     }
 
+    /**
+     * @return null
+     */
+    protected function getDefaultLanguageId()
```

Comment:
You can retrieve the default language using the following code:
`Configuration::get('PS_LANG_DEFAULT', null, $shop->id_shop_group, $shop->id)`